<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtDLGNanomiteClass</class>
 <widget class="QMainWindow" name="qtDLGNanomiteClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>798</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>775</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[Nanomite v 0.1] - MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtDLGNanomite.qrc">
    <normaloff>:/qtDLGNanomite/Icons/IDI_MAIN.ico</normaloff>:/qtDLGNanomite/Icons/IDI_MAIN.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tblLogBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>770</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>14</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>14</number>
    </attribute>
    <column>
     <property name="text">
      <string>Time Stamp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Log String</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tblStack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>380</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>14</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>14</number>
    </attribute>
    <column>
     <property name="text">
      <string>Offset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comment</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tblCallstack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>1170</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>14</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>14</number>
    </attribute>
    <column>
     <property name="text">
      <string>Stack Offset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Func. Addr</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>&lt;mod.func&gt;</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Return To</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Return To - &lt;mod.func&gt;</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source Line</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source File</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tblDisAs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>810</width>
      <height>380</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>14</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>14</number>
    </attribute>
    <column>
     <property name="text">
      <string>Offset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OpCodes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mnemonics</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comment</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tblRegView">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>340</width>
      <height>380</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>15</number>
    </attribute>
    <column>
     <property name="text">
      <string>Register</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QScrollBar" name="scrollStackView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>541</y>
      <width>20</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="scrollDisAs">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>20</width>
      <height>380</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_OpenNew"/>
    <addaction name="actionFile_AttachTo"/>
    <addaction name="actionFile_Detach"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebug_Start"/>
    <addaction name="actionDebug_Suspend"/>
    <addaction name="actionDebug_Stop"/>
    <addaction name="actionDebug_Restart"/>
    <addaction name="separator"/>
    <addaction name="actionDebug_Step_In"/>
    <addaction name="actionDebug_Step_Over"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions_Options"/>
    <addaction name="separator"/>
    <addaction name="actionOptions_About"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionWindow_Detail_Information"/>
    <addaction name="actionWindow_Breakpoint_Manager"/>
    <addaction name="separator"/>
    <addaction name="actionWindow_Show_Memory"/>
    <addaction name="actionWindow_Show_Heap"/>
    <addaction name="actionWindow_Show_Strings"/>
    <addaction name="actionWindow_Show_Debug_Output"/>
    <addaction name="actionWindow_Show_Handles"/>
    <addaction name="actionWindow_Show_Windows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menuWindows"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDebug_Start"/>
   <addaction name="actionDebug_Stop"/>
   <addaction name="actionDebug_Suspend"/>
   <addaction name="actionDebug_Restart"/>
   <addaction name="actionDebug_Step_In"/>
   <addaction name="actionDebug_Step_Over"/>
   <addaction name="separator"/>
   <addaction name="actionWindow_Detail_Information"/>
   <addaction name="actionWindow_Breakpoint_Manager"/>
   <addaction name="separator"/>
   <addaction name="actionWindow_Show_Memory"/>
   <addaction name="actionWindow_Show_Heap"/>
   <addaction name="actionWindow_Show_Strings"/>
   <addaction name="actionWindow_Show_Debug_Output"/>
   <addaction name="actionWindow_Show_Handles"/>
   <addaction name="actionWindow_Show_Windows"/>
  </widget>
  <widget class="QStatusBar" name="stateBar"/>
  <action name="actionFile_OpenNew">
   <property name="text">
    <string>Open new File</string>
   </property>
  </action>
  <action name="actionFile_AttachTo">
   <property name="text">
    <string>Attach To</string>
   </property>
  </action>
  <action name="actionFile_Detach">
   <property name="text">
    <string>Detach</string>
   </property>
  </action>
  <action name="actionFile_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDebug_Start">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_START.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_START.ico</normalon>:/qtDLGNanomite/IDI_START.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionDebug_Stop">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_STOP.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_STOP.ico</normalon>:/qtDLGNanomite/IDI_STOP.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDebug_Suspend">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_SUSPEND.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_SUSPEND.ico</normalon>:/qtDLGNanomite/IDI_SUSPEND.ico</iconset>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
  </action>
  <action name="actionDebug_Restart">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_RESTART.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_RESTART.ico</normalon>:/qtDLGNanomite/IDI_RESTART.ico</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionDebug_Step_In">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_STEPIN.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_STEPIN.ico</normalon>:/qtDLGNanomite/IDI_STEPIN.ico</iconset>
   </property>
   <property name="text">
    <string>Step In</string>
   </property>
  </action>
  <action name="actionDebug_Step_Over">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/IDI_STEPOVER.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/IDI_STEPOVER.ico</normalon>:/qtDLGNanomite/IDI_STEPOVER.ico</iconset>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
  </action>
  <action name="actionOptions_Options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionOptions_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWindow_Detail_Information">
   <property name="icon">
    <iconset>
     <normaloff>:/qtDLGNanomite/ID_showInfo.ico</normaloff>
     <normalon>:/qtDLGNanomite/Icons/ID_showInfo.ico</normalon>:/qtDLGNanomite/ID_showInfo.ico</iconset>
   </property>
   <property name="text">
    <string>Detail Information</string>
   </property>
  </action>
  <action name="actionWindow_Breakpoint_Manager">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_BPManager.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Breakpoint Manager</string>
   </property>
  </action>
  <action name="actionWindow_Show_Memory">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_Memory.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Memory</string>
   </property>
  </action>
  <action name="actionWindow_Show_Heap">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_Heap.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Heap</string>
   </property>
  </action>
  <action name="actionWindow_Show_Strings">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_Strings.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Strings</string>
   </property>
  </action>
  <action name="actionWindow_Show_Debug_Output">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_DebugString.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Debug Output</string>
   </property>
  </action>
  <action name="actionWindow_Show_Handles">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_Handles.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Handles</string>
   </property>
  </action>
  <action name="actionWindow_Show_Windows">
   <property name="icon">
    <iconset>
     <normalon>:/qtDLGNanomite/Icons/ID_Windows.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Windows</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tblDisAs</tabstop>
  <tabstop>tblRegView</tabstop>
  <tabstop>tblCallstack</tabstop>
  <tabstop>tblStack</tabstop>
  <tabstop>tblLogBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtDLGNanomite.qrc"/>
 </resources>
 <connections/>
</ui>
